tests/configuration/fixtures.py

```diff
@@ -4,6 +4,6 @@
 
 @pytest.fixture
 def mixpanel_client():
-    mixpanel_client = MixpanelClient('API_SECRET', api_domain="mixpanel.com")
+    mixpanel_client = MixpanelClient('API_SECRET', api_domain="mixpanel.com", request_timeout=1) # Pass extra request_timeout parameter
```

Comment:
Can you also add the reason of passing extra param in the comment?

Reply:
As support of request_timeout has been added, MixpanelClient accepts the request_timeout parameter which is mandatory.
Added comment in code also.